cmake/Dependencies.cmake

```diff
@@ -839,6 +839,12 @@ else()
             DESTINATION ${CMAKE_INSTALL_PREFIX}
             FILES_MATCHING PATTERN "*.h")
 endif()
+if (MSVC)
+  if ((GENERATOR_IS_MULTI_CONFIG AND "Debug" IN_LIST CMAKE_CONFIGURATION_TYPES)
+      OR ${CMAKE_BUILD_TYPE} STREQUAL "Debug")
+    add_compile_definitions(Py_DEBUG)
```

Comment:
This is not needed. When `_DEBUG` is defined, `Py_DEBUG` is defined too. The logic is in `pyconfig.h`.

Reply:
> This is not needed. When _DEBUG is defined, Py_DEBUG is defined too. The logic is in pyconfig.h.

pybind11 undef _DEBUG deliberately

Reply:
> This is not needed. When _DEBUG is defined, Py_DEBUG is defined too. The logic is in pyconfig.h.
> 
> pybind11 undef _DEBUG deliberately

see this https://github.com/pybind/pybind11/issues/1295

Reply:
Okay.

Reply:
> Okay.

I use this macro for system pybind11 and for the updated 3rd module in future .